components/relevancy/src/relevancy.udl

```diff
@@ -22,7 +22,7 @@ interface RelevancyStore {
 
     // Ingest the top URLs by frequency to build up the user's interest vector
     [Throws=RelevancyApiError]
-    void ingest(sequence<string> top_urls);
+    InterestVector ingest(sequence<string> top_urls);
```

Comment:
For the experiment, I think it's fine to return an interest vector.  Eventually I think we want to store this data in the DB and incrementally update it when we get new URLs.  For now, how about keeping the code the same and adding a comment about our plans.